server/utils/utils.ts

```diff
@@ -197,12 +198,34 @@ const selectReleaseDate = (nomisRecord: Prisoner) => {
   return dateString
 }
 
-const isReleaseDateBeforeCutOffDate = (cutOffDate: string, releaseDate: string): boolean => {
+const isAttentionNeeded = (licence: ManagedCaseLicence, nomisRecord: Prisoner) => {
+  const today = new Date()
+  const licenceStartDate = licence.licenceStartDate ? parse(licence?.licenceStartDate, 'dd/MM/yyyy', new Date()) : ''
+  const status = ['APPROVED', 'SUBMITTED', 'IN_PROGRESS', 'NOT_STARTED']
+  if (
+    (status.includes(licence?.status) && !nomisRecord.confirmedReleaseDate && !nomisRecord.conditionalReleaseDate) || // If licence status is ‘approved’, ‘submitted’, ‘in progress' or 'not started’ AND there is no CRD/ARD
+    (status[0].includes(licence?.status) && isBefore(licenceStartDate, today)) // If licence status is ‘approved’ AND CRD/ARD is in the past(licenceStartDate is equalto ARD/CRD)
+  ) {
+    return true
+  }
+  return false
+}
+
+const isReleaseDateOnOrBeforeCutOffDate = (cutOffDate: string, releaseDate: string): boolean => {
   const rDate = parse(releaseDate, 'dd/MM/yyyy', new Date())
   const cDate = parse(cutOffDate, 'dd/MM/yyyy', new Date())
   return isBefore(rDate, cDate) || isEqual(rDate, cDate)
 }
 
+const caseTabType = (licence: ManagedCaseLicence, nomisRecord: Prisoner, cutOffDate: string) => {
```

Comment:
rather than return a string you could define a `ComCaseTabType` union: `attentionNeeded | releasesInNextTwoWorkingDays | futureReleases` .

Which would give you more compile safety.. or an enum?

Reply:
could call this `determineComCreateCasesTab`